portfolio/src/main/webapp/script.js

```diff
@@ -56,7 +57,84 @@ function getRandomFile(element, path, fileName, numElems) {
  * @param {DOM element} element - an element in the DOM for selecting
  */
 function fetchAndModify(path, element) {
-  fetch(path).then(response => response.text()).then(content => 
-    element.innerHTML= content
-  )
+  fetch(path)
+      .then(response => response.text())
+      .then(content => element.innerHTML = content)
 }
+/*
+ * This code creates an innerHTML and then reads its text to remove special
```

Comment:
Remove "This code" and just start with "Creates an ..." Same for the other new functions.

https://google.github.io/styleguide/jsguide.html#jsdoc-method-and-function-comments